src/main/java/dto/PersonDTO_IN.java

```diff
@@ -11,19 +12,19 @@
 @Schema(name = "Person")
 public class PersonDTO_IN {
 
-    @Schema(name = "ID", required = true, example = "1")
+    @Schema(description = "Unique Identifier", required = true, example = "1")
     private Integer id;
-    @Schema(name = "First Name", required = true, example = "Johnny")
+    @Schema(description = "First Name", required = true, example = "Johnny")
     private String firstName;
-    @Schema(name = "Last Name", required = true, example = "Reimar")
+    @Schema(description = "Last Name", required = true, example = "Reimar")
     private String lastName;
-    @Schema(name = "Email", required = true, example = "Johnny@Reimar.dk")
+    @Schema(description = "Email", required = true, example = "Johnny@Reimar.dk")
     private String email;
-    @Schema(example = "None available")
+    @Schema(example = "{\"name\":\"Lacrosse\",\"description\":\"Rich people game\"}")
     private List<HobbyDTO_IN> hobbies = new ArrayList();
-    @Schema(example = "None available")
+    @Schema(example = "{\"number\":\"12345678\",\"description\":\"work\"}")
     private List<PhoneDTO_IN> phones = new ArrayList();
-    @Schema(name = "Address", example = "None available")
+    @Schema(example = "[{\"street\":\"Vejnavn\",\"additionalInfo\":\"Nr. 12, 1tv\",\"cityInfo\":{\"zipCode\":\"2800\",\"city\":\"Lyngby\"}}]")
```

Comment:
>Manually doing JSON
>:kms:

Reply:
tell me about it.

I even had it prettyfied but it does **NOT** like that